File: Data Structures & Algorithms/rotate-array/submission-1.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        l = 0
        r = len(nums)-k

        while r < len(nums) :
            nums[l],nums[r] = nums[r],nums[l]
            l+=1
            r+=1

        

File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map < int , int > count;
        int max = 0 , max_count = 0;

        for ( int num : nums){
            count[num]++;
            if (count[num] > max_count){
                max = num;
                max_count = count[num];
            }
        }
        return max;
        

    }
};